A Clock With Benefits

tomburtonwood opened the pull request:
i'm worried my proposals might be too complex for the actual project. but these are the top four projects i want to work on over the semester.

bakercp commented:
Hey @tomburtonwood this looks great.  A few little website structure things ... since you're the first, let's move the images folder to the root of the website -- you can do that by changing the directory to the `_posts` folder and running:

`git mv images/ ../` 

That will move the images to the root folder.

Then, when you want to reference them in a post, use the following code:

```html
<img src="{{ site.baseurl }}/images/enzo.png" width="600px" />
```

or for pure markdown (if you don't care about image size).

```md
![]({{ site.baseurl }}/images/enzo.png)
```

The `{{ site.baseurl }}` will grab the base URL from the config and fill it in correctly.

tomburtonwood commented:
cool - yeah i was having a hard time getting the image markdown code to
work - but in retrospect scaling the imgs down is prob a good idea, though
i guess i could reformat them in terminal - so .... maybe i'll convert to
markdown after all.



On Sun, Sep 13, 2015 at 8:31 PM, Christopher Baker <notifications@github.com
> wrote:

> Hey @tomburtonwood <https://github.com/tomburtonwood> this looks great. A
> few little website structure things ... since you're the first, let's move
> the images folder to the root of the website -- you can do that by changing
> the directory to the _posts folder and running:
>
> git mv images/ ../
>
> That will move the images to the root folder.
>
> Then, when you want to reference them in a post, use the following code:
>
> <img src="{{ site.baseurl }}/images/enzo.png" width="600px" />
>
> or for pure markdown (if you don't care about image size).
>
> ![]({{ site.baseurl }}/images/enzo.png)
>
> The {{ site.baseurl }} will grab the base URL from the config and fill it
> in correctly.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bakercp/InteractiveArt2015/pull/21#issuecomment-139936862>
> .
>



-- 

tom burtonwood
tburtonwood@gmail.com

http://tomburtonwood.com
http://learn3dprinting.co

phone: 773-837-9809
twitter: @tburtonwood
Skype: tburtonwood
instagram: @tomburtonwood

Need 3D Printing? Check out our 3D Hub:
https://www.3dhubs.com/chicago/hubs/simile

bakercp commented:
If you want to resize images on the command line -- just for the fun of it, `sips` or image magick are good (http://www.maclife.com/article/columns/terminal_101_resize_images_terminal).  Smaller images in git repositories are usually preferred.

bakercp commented:
Ahh, one other thing -- you need to prefix your post filename with the date.  You can fix yours like this if you are in the `_posts` directory:

```
git mv Burtonwood-ClockWithBenefits.md 2015-09-13-Burtonwood-ClockWithBenefits.md
```

tomburtonwood commented:
will do. i tried to use git last year to publish my last 3DP book and i
uploaded, or tried to, probably 100mb in STL's did not work very well :(

On Sun, Sep 13, 2015 at 8:37 PM, Christopher Baker <notifications@github.com
> wrote:

> If you want to resize images on the command line -- just for the fun of
> it, sips or image magick are good (
> http://www.maclife.com/article/columns/terminal_101_resize_images_terminal).
> Smaller images in git repositories are usually preferred.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bakercp/InteractiveArt2015/pull/21#issuecomment-139937729>
> .
>



-- 

tom burtonwood
tburtonwood@gmail.com

http://tomburtonwood.com
http://learn3dprinting.co

phone: 773-837-9809
twitter: @tburtonwood
Skype: tburtonwood
instagram: @tomburtonwood

Need 3D Printing? Check out our 3D Hub:
https://www.3dhubs.com/chicago/hubs/simile

tomburtonwood commented:
got it. i did not know if that was auto generated.


On Sun, Sep 13, 2015 at 8:39 PM, Christopher Baker <notifications@github.com
> wrote:

> Ahh, one other thing -- you need to prefix your post filename with the
> date. You can fix yours like this if you are in the _posts directory:
>
> git mv Burtonwood-ClockWithBenefits.md 2015-09-13-Burtonwood-ClockWithBenefits.md
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bakercp/InteractiveArt2015/pull/21#issuecomment-139937852>
> .
>



-- 

tom burtonwood
tburtonwood@gmail.com

http://tomburtonwood.com
http://learn3dprinting.co

phone: 773-837-9809
twitter: @tburtonwood
Skype: tburtonwood
instagram: @tomburtonwood

Need 3D Printing? Check out our 3D Hub:
https://www.3dhubs.com/chicago/hubs/simile

tomburtonwood commented:
is there a way to git add all updated files automatically rather than
typing in all the files names?

On Sun, Sep 13, 2015 at 8:40 PM, tom burtonwood <tburtonwood@gmail.com>
wrote:

> got it. i did not know if that was auto generated.
>
>
> On Sun, Sep 13, 2015 at 8:39 PM, Christopher Baker <
> notifications@github.com> wrote:
>
>> Ahh, one other thing -- you need to prefix your post filename with the
>> date. You can fix yours like this if you are in the _posts directory:
>>
>> git mv Burtonwood-ClockWithBenefits.md 2015-09-13-Burtonwood-ClockWithBenefits.md
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/bakercp/InteractiveArt2015/pull/21#issuecomment-139937852>
>> .
>>
>
>
>
> --
>
> tom burtonwood
> tburtonwood@gmail.com
>
> http://tomburtonwood.com
> http://learn3dprinting.co
>
> phone: 773-837-9809
> twitter: @tburtonwood
> Skype: tburtonwood
> instagram: @tomburtonwood
>
> Need 3D Printing? Check out our 3D Hub:
> https://www.3dhubs.com/chicago/hubs/simile
>
>


-- 

tom burtonwood
tburtonwood@gmail.com

http://tomburtonwood.com
http://learn3dprinting.co

phone: 773-837-9809
twitter: @tburtonwood
Skype: tburtonwood
instagram: @tomburtonwood

Need 3D Printing? Check out our 3D Hub:
https://www.3dhubs.com/chicago/hubs/simile

tomburtonwood commented:
not sure why imgs are not showing up now
https://github.com/tomburtonwood/InteractiveArt2015/blob/gh-pages/_posts/2015-09-13-Burtonwood-ClockWithBenefits.md

i changed images to _images and changed the markdown to reflect this - but
nope.


On Sun, Sep 13, 2015 at 8:49 PM, tom burtonwood <tburtonwood@gmail.com>
wrote:

> is there a way to git add all updated files automatically rather than
> typing in all the files names?
>
> On Sun, Sep 13, 2015 at 8:40 PM, tom burtonwood <tburtonwood@gmail.com>
> wrote:
>
>> got it. i did not know if that was auto generated.
>>
>>
>> On Sun, Sep 13, 2015 at 8:39 PM, Christopher Baker <
>> notifications@github.com> wrote:
>>
>>> Ahh, one other thing -- you need to prefix your post filename with the
>>> date. You can fix yours like this if you are in the _posts directory:
>>>
>>> git mv Burtonwood-ClockWithBenefits.md 2015-09-13-Burtonwood-ClockWithBenefits.md
>>>
>>> —
>>> Reply to this email directly or view it on GitHub
>>> <https://github.com/bakercp/InteractiveArt2015/pull/21#issuecomment-139937852>
>>> .
>>>
>>
>>
>>
>> --
>>
>> tom burtonwood
>> tburtonwood@gmail.com
>>
>> http://tomburtonwood.com
>> http://learn3dprinting.co
>>
>> phone: 773-837-9809
>> twitter: @tburtonwood
>> Skype: tburtonwood
>> instagram: @tomburtonwood
>>
>> Need 3D Printing? Check out our 3D Hub:
>> https://www.3dhubs.com/chicago/hubs/simile
>>
>>
>
>
> --
>
> tom burtonwood
> tburtonwood@gmail.com
>
> http://tomburtonwood.com
> http://learn3dprinting.co
>
> phone: 773-837-9809
> twitter: @tburtonwood
> Skype: tburtonwood
> instagram: @tomburtonwood
>
> Need 3D Printing? Check out our 3D Hub:
> https://www.3dhubs.com/chicago/hubs/simile
>
>


-- 

tom burtonwood
tburtonwood@gmail.com

http://tomburtonwood.com
http://learn3dprinting.co

phone: 773-837-9809
twitter: @tburtonwood
Skype: tburtonwood
instagram: @tomburtonwood

Need 3D Printing? Check out our 3D Hub:
https://www.3dhubs.com/chicago/hubs/simile

bakercp commented on _posts/2015-09-13-Burtonwood-ClockWithBenefits.md:
```diff
@@ -0,0 +1,31 @@
+author: tomburtonwood
+---
+
+![]({{ site.baseurl }}_images/enzo.png)
```
I believe these should be:

```
![]({{ site.baseurl }}/images/enzo.png)
```

bakercp replied:
Also you'll need to put them in a folder called `images` rather than `_images`.

bakercp commented:
Also, can you put them in a folder called "images" rather than "_images".  I think the underscore causes them to not get added to the site as assets.

bakercp commented:
Regarding adding all changes ... generally it's a good idea to be specific, though you can use wildcards like *.  The problem there is that it will often add stuff you don't want to ... like `.DS_Store` files, etc if you aren't ignoring them with the `.gitignore` file.

bakercp commented on _data/people/tomburtonwood.yml.save:
```diff
@@ -0,0 +1,8 @@
+user: tomburtonwood
```
I think this file should be removed ...

tomburtonwood replied:
eeek. sorry. my bad.

On Sun, Sep 13, 2015 at 9:34 PM, Christopher Baker <notifications@github.com
> wrote:

> In _data/people/tomburtonwood.yml.save
> <https://github.com/bakercp/InteractiveArt2015/pull/21#discussion_r39357643>
> :
>
> > @@ -0,0 +1,8 @@
> > +user: tomburtonwood
>
> I think this file should be removed ...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bakercp/InteractiveArt2015/pull/21/files#r39357643>.
>



-- 

tom burtonwood
tburtonwood@gmail.com

http://tomburtonwood.com
http://learn3dprinting.co

phone: 773-837-9809
twitter: @tburtonwood
Skype: tburtonwood
instagram: @tomburtonwood

Need 3D Printing? Check out our 3D Hub:
https://www.3dhubs.com/chicago/hubs/simile

bakercp replied:
Note that it is ... `yml.save`

bakercp commented on _data/people/tomburtonwood.yml.save.1:
```diff
@@ -0,0 +1,8 @@
+user: tomburtonwood
```
I think this file should be removed too ...

bakercp replied:
Note that it is ... `yml.save.1`

tomburtonwood replied:
ok i think everything is back to as it should be.

imgs still not showing up though. -

On Sun, Sep 13, 2015 at 9:35 PM, Christopher Baker <notifications@github.com
> wrote:

> In _data/people/tomburtonwood.yml.save.1
> <https://github.com/bakercp/InteractiveArt2015/pull/21#discussion_r39357669>
> :
>
> > @@ -0,0 +1,8 @@
> > +user: tomburtonwood
>
> Note that it is ... yml.save.1
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bakercp/InteractiveArt2015/pull/21/files#r39357669>.
>



-- 

tom burtonwood
tburtonwood@gmail.com

http://tomburtonwood.com
http://learn3dprinting.co

phone: 773-837-9809
twitter: @tburtonwood
Skype: tburtonwood
instagram: @tomburtonwood

Need 3D Printing? Check out our 3D Hub:
https://www.3dhubs.com/chicago/hubs/simile

bakercp commented on _posts/2015-09-13-Burtonwood-ClockWithBenefits.md:
```diff
@@ -0,0 +1,31 @@
+
+software/app: To build my own scanner using a kinect / structure, to try to improve the resolution  and get better results, either larger scanned areas, increasing the focal length, to scan from a drone in realtime for example, or to scan larger structures with better resolution results, than currently available
+
+![]({{ site.baseurl }} /images/sullivan.JPG)
```
remove the space between `}} /images`

bakercp commented on _posts/2015-09-13-Burtonwood-ClockWithBenefits.md:
```diff
@@ -0,0 +1,31 @@
+
+software/app: Augmented Reality Application. For upcoming Sullivan book, i want to make an AR app to provide access to extra layers on content that cannot easily be “displayed” on the 3D printed or physical pages. This comes out of a long standing interest in embedding scannable symbols or trackers in 3d printed or digitally fabricated pieces - it would be awesome to figure out a way to do this i ...
+
+![]({{ site.baseurl }} /images/madlion-mold.png)
```
again, remove the space.  should look like:

`![]({{ site.baseurl }}/images/madlion-mold.png)`

bakercp commented:
So close!

tomburtonwood commented:
sorry. i missed those. they were my tests. are you seeing any of my images
at that end? i don't see any at this end. will update.

On Sun, Sep 13, 2015 at 10:10 PM, Christopher Baker <
notifications@github.com> wrote:

> So close!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bakercp/InteractiveArt2015/pull/21#issuecomment-139949574>
> .
>



-- 

tom burtonwood
tburtonwood@gmail.com

http://tomburtonwood.com
http://learn3dprinting.co

phone: 773-837-9809
twitter: @tburtonwood
Skype: tburtonwood
instagram: @tomburtonwood

Need 3D Printing? Check out our 3D Hub:
https://www.3dhubs.com/chicago/hubs/simile

bakercp commented:
Looks like it's all working now ...

http://tomburtonwood.github.io/InteractiveArt2015/post/2015/09/12/Burtonwood-ClockWithBenefits.html

tomburtonwood commented:
aha! i'm looking at my github page - the imgs are not showing there. i did
not think to look on the class website. any thoughts why they show up
there, but not here


https://github.com/tomburtonwood/InteractiveArt2015/blob/gh-pages/_posts/2015-09-13-Burtonwood-ClockWithBenefits.md
?


On Sun, Sep 13, 2015 at 10:17 PM, Christopher Baker <
notifications@github.com> wrote:

> Looks like it's all working now ...
>
>
> http://tomburtonwood.github.io/InteractiveArt2015/post/2015/09/12/Burtonwood-ClockWithBenefits.html
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bakercp/InteractiveArt2015/pull/21#issuecomment-139951402>
> .
>



-- 

tom burtonwood
tburtonwood@gmail.com

http://tomburtonwood.com
http://learn3dprinting.co

phone: 773-837-9809
twitter: @tburtonwood
Skype: tburtonwood
instagram: @tomburtonwood

Need 3D Printing? Check out our 3D Hub:
https://www.3dhubs.com/chicago/hubs/simile

tomburtonwood commented:
thank you for your help btw!

On Sun, Sep 13, 2015 at 10:19 PM, tom burtonwood <tburtonwood@gmail.com>
wrote:

> aha! i'm looking at my github page - the imgs are not showing there. i did
> not think to look on the class website. any thoughts why they show up
> there, but not here
>
>
> https://github.com/tomburtonwood/InteractiveArt2015/blob/gh-pages/_posts/2015-09-13-Burtonwood-ClockWithBenefits.md
> ?
>
>
> On Sun, Sep 13, 2015 at 10:17 PM, Christopher Baker <
> notifications@github.com> wrote:
>
>> Looks like it's all working now ...
>>
>>
>> http://tomburtonwood.github.io/InteractiveArt2015/post/2015/09/12/Burtonwood-ClockWithBenefits.html
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/bakercp/InteractiveArt2015/pull/21#issuecomment-139951402>
>> .
>>
>
>
>
> --
>
> tom burtonwood
> tburtonwood@gmail.com
>
> http://tomburtonwood.com
> http://learn3dprinting.co
>
> phone: 773-837-9809
> twitter: @tburtonwood
> Skype: tburtonwood
> instagram: @tomburtonwood
>
> Need 3D Printing? Check out our 3D Hub:
> https://www.3dhubs.com/chicago/hubs/simile
>
>


-- 

tom burtonwood
tburtonwood@gmail.com

http://tomburtonwood.com
http://learn3dprinting.co

phone: 773-837-9809
twitter: @tburtonwood
Skype: tburtonwood
instagram: @tomburtonwood

Need 3D Printing? Check out our 3D Hub:
https://www.3dhubs.com/chicago/hubs/simile

bakercp commented:
Yeah, they don't show up there because the correct links are not generated until it gets turned into a website on upload.  Note that http://tomburtonwood.github.io/InteractiveArt2015/post/2015/09/12/Burtonwood-ClockWithBenefits.html is _your_ version of the site, not the class'.  Now I'll pull your changes in and you will see your additions in the overall class site.

http://bakercp.github.io/InteractiveArt2015/post/2015/09/12/Burtonwood-ClockWithBenefits.html

bakercp commented:
Thanks!

tomburtonwood commented:
awesome. thank you - i wanted to get everything online sooner than later,
tomorrow is week 1 of my sculpture class and i still have a lecture to
finish for Weds night. i am sorry to miss Weds class, but i'm hoping to do
the ofSketch graphics work tomorrow or tues night.

t

On Sun, Sep 13, 2015 at 10:21 PM, Christopher Baker <
notifications@github.com> wrote:

> Thanks!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bakercp/InteractiveArt2015/pull/21#issuecomment-139951674>
> .
>



-- 

tom burtonwood
tburtonwood@gmail.com

http://tomburtonwood.com
http://learn3dprinting.co

phone: 773-837-9809
twitter: @tburtonwood
Skype: tburtonwood
instagram: @tomburtonwood

Need 3D Printing? Check out our 3D Hub:
https://www.3dhubs.com/chicago/hubs/simile

bakercp commented:
Super.  Thanks for your hard work getting all of that up.

tomburtonwood commented:
hey - that img resize link u sent is great. for years i've been wanting to
do that without having to open preview or psd - now i want to make an app
to tag and organize all my photos as i take them on my phone.

On Sun, Sep 13, 2015 at 10:28 PM, Christopher Baker <
notifications@github.com> wrote:

> Super. Thanks for your hard work getting all of that up.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bakercp/InteractiveArt2015/pull/21#issuecomment-139952787>
> .
>



-- 

tom burtonwood
tburtonwood@gmail.com

http://tomburtonwood.com
http://learn3dprinting.co

phone: 773-837-9809
twitter: @tburtonwood
Skype: tburtonwood
instagram: @tomburtonwood

Need 3D Printing? Check out our 3D Hub:
https://www.3dhubs.com/chicago/hubs/simile
